templates/arm/stm32f4cube/mods.config

```diff
@@ -50,13 +50,13 @@ configuration conf {
 	include embox.init.setup_tty_diag
 	@Runlevel(3) include embox.init.start_script(shell_name="diag_shell")
 
-	include embox.cmd.fs.ls
+	include embox.cmd.fs.ls /*
```

Comment:
See my remark about generated object files' sizes. I don't know another way to make template compilable with clang yet.